recipes/pulsar_phase/pulsar_phase_computation.ipynb

```diff
@@ -0,0 +1,1895 @@
+{
```

Comment:
<p>I would recommend to insist on the metadata, either with a paragraph title in bold or in the title of the section 3 or something else...</p><p> </p><p>Also, I would mention that there is not yet a 'standard' format for these metadata (in GADF or elsewhere). So it is worth to precise that this is an ad-hoc format that contains the most useful information to keep track of the provenance.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy-recipes/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>